scripts/deploy-artifacts

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Deploy a jar, source jar, and javadoc jar to Sonatype's snapshot repo.
+#
+# Adapted from https://coderwall.com/p/9b_lfq and
+# http://benlimmer.com/2013/12/26/automatically-publish-javadoc-to-gh-pages-with-travis-ci/
+
+BRANCH="master"
+
+set -e
+
+if [ "${GITHUB_REF##*/}" != "$BRANCH" ]; then
+  echo "Skipping snapshot deployment: wrong branch. Expected '$BRANCH' but was '${GITHUB_REF##*/}'."
+else
+  echo "Deploying dropbox-sdk-java..."
+  openssl aes-256-cbc -md sha256 -d -in scripts/sdk-sign-secret.gnupg-keyring.aes -out scripts/sdk-sign-secret.gnupg-keyring -k "${ENCRYPT_KEY}"
```

Comment:
Not a bash expert, but the paths here look suspicious. Can you confirm that we don't need any `${PWD}` refs in this line when passing `scripts/sdk-sign-secret.gnupg-keyring.aes` as an input (and the other `scripts` param)?

Reply:
Locally this worked okay, we'll see what github says when we land.